src/cpp/session/modules/SessionThemes.R

```diff
@@ -936,13 +980,25 @@
 
    if (!file.remove(themeList[[name]]$fileName))
    {
-      # TODO: give better error when the theme is installed globally.
-      if (file.exists(theme[[name]]$fileName))
+      if (file.exists(themeList[[name]]$fileName))
       {
-         stop(
-            "Unable to remove the speicified theme \"",
-            name,
-            "\". Please check your file system permissions.")
+         justFileName <- basename(themeList[[name]]$fileName)
+         actualPath <- normalizePath(themeList[[name]]$fileName)
+         globalPath <- normalizePath(.rs.getThemeInstallDir(TRUE))
+         if (file.path(globalPath, justFileName) == actualPath)
```

Comment:
We typically use `identical(x, y)` when comparing strings for equality since it can tolerate unusual values (nulls, zero-length vectors, etc.)